src/screens/organizations/organizations.css

```diff
@@ -1,4 +1,3 @@
-/* organizations */
 .organizations {
```

Comment:
Let's create a `backgroundImage` in the tailwind config for this, instead of custom CSS.

Reply:
I meant to at first, but didn't find a way to use multiple background URLs on the same element (I mean we can use arbitrary values [like this](https://github.com/tailwindlabs/tailwindcss/discussions/7444) but it's a bit awkward).

Or maybe we can "cheat" using dummy wrappers but that wouldn't be too pretty either.

Reply:
The way I did this with other screens, was have different absolute positioned images placed in front. I think this makes it a bit clearer what image overlays the other.

Reply:
I did that, except I still used backgrounds instead of images (as it was easier to position them this way). I think it's fine but please lmk.